internal/acceptance/mws_network_connectivity_config_test.go

```diff
@@ -0,0 +1,38 @@
+package acceptance
+
+import (
+	"os"
+	"testing"
+)
+
+func TestAzureAccNetworkConnectivityConfig(t *testing.T) {
+	cloudEnv := os.Getenv("CLOUD_ENV")
+	switch cloudEnv {
+	case "azure":
```

Comment:
This should be "azure-ucacct" (confusingly).

Reply:
used new function